index.js

```diff
@@ -167,23 +167,25 @@
 //. silent failures due to type coercion (at worst). For example:
 //.
 //. ```javascript
-//. S.inc('XXX');
+//. S.add(2, true);
 //. // ! TypeError: Invalid value
 //. //
-//. //   inc :: FiniteNumber -> FiniteNumber
-//. //          ^^^^^^^^^^^^
-//. //               1
+//. //   add :: FiniteNumber -> FiniteNumber -> FiniteNumber
+//. //                          ^^^^^^^^^^^^
+//. //                               1
 //. //
-//. //   1)  "XXX" :: String
+//. //   1)  true :: Boolean
 //. //
 //. //   The value at position 1 is not a member of ‘FiniteNumber’.
+//. //
+//. //   See https://github.com/sanctuary-js/sanctuary-def/tree/v0.11.0#FiniteNumber for information about the sanctuary-def/FiniteNumber type.
```

Comment:
I don't like having to remember to update this each time we upgrade the sanctuary-def dependency.